etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/hapi/HapiOrderConverter.java

```diff
@@ -105,6 +106,40 @@ public Order<?> convertMetadataToOmlOrder(Order<?> order) {
         return new HapiOrder(orderBundle);
     }
 
+    @Override
+    public Order<?> addContactSectionToPatientResource(Order<?> order) {
+        logger.logInfo("Adding contact section in Patient resource");
+
+        var hapiOrder = (Order<Bundle>) order;
+        var orderBundle = hapiOrder.getUnderlyingOrder();
+        var codingList = new ArrayList<Coding>();
+        codingList.add(
+                new Coding()
+                        .setSystem("http://terminology.hl7.org/CodeSystem/v3-RoleCode")
+                        .setCode("MTH")
+                        .setDisplay("mother"));
+
+        HapiHelper.resourcesInBundle(orderBundle, Patient.class)
+                .forEach(
+                        p -> {
+                            var myContact = p.addContact();
+                            myContact.addRelationship();
+                            myContact.getRelationshipFirstRep().setCoding(codingList);
```

Comment:
`myContact.addRelationship()` returns the relationship object.  You could assign it to a variable and use that instead of `myContact.getRelationshipFirstRep()`.

For example...
```suggestion
                            var motherRelationship = myContact.addRelationship();
                            motherRelationship.setCoding(codingList);
```